### ISSUE TITLE
ASCO optimization doesn't work with QucsatorRF but works with Qucsator

### ISSUE THREAD

**Issue by ra3xdh** [ra3xdh/qucs_s#830]
The reason for this issue is that `qucsator` executable name is hardcoded inside ASCO sources. Also ASCO always assumes that `qucsator` is available from $PATH. This may make difficulties for running a portable version. The ways to resolve this:

* Provide a patched version of ASCO with the next release
* Submit a patch to ASCO that allows to set the path to qucsator excutable

**Comment by ra3xdh** [comment]
I have prepared a patched version of ASCO here: https://github.com/ra3xdh/asco_qucs_s Compile and install this version if you need to run optimizer with Qucs-S+QucsatorRF. You have to set the environment variable `ASCO_SIM_PATH` to full path to `qucsator_rf` if you are using stable v24.2.1 release. For example `ASCO_SIM_PATH=/usr/bin/qucsator_rf` The version from Git applies this environment variable automatically. I have submitted a patch to ASCO developers. Let's wait for reaction. If the patch will be not accepted, I will provide the patched ASCO version with the next Qucs-S release.

**Comment by ra3xdh** [comment]
I have contacted ASCO developers and they didn't advice to patch the ASCO. The #843 provides an another solution. 

* For Linux the temporary symlink `qucsator` pointing to `qucsator_rf` is created. This makes possible to run the system-wide installed vanilla ASCO and avoid the distribution of the patched ASCO version with DEB/RPM packages. This prevents a possible conflict with ASCO package on some distributions. 
* For Windows only patched ASCO version works. The approach using `general.sh` didn't work because the script name again is hardcoded inside ASCO. I will add a patched `asco.exe` in the Windows installer. This is not a problem for Windows. 

Closing this as resolved.
